content/blog/getting-started-with-volume-snapshots-on-a-kubernetes-cluster-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,840 @@
+---
+title: Getting started with volume snapshots on a Kubernetes cluster in HPE
+  GreenLake for Private Cloud Enterprise
+date: 2024-01-23T14:24:28.483Z
+author: Guoping Jia
+authorimage: /img/guoping.png
+disable: false
+tags:
+  - Kubernetes
+  - Persistent volumes
+  - Volume snapshots
+  - HPE CSI driver for Kubernetes
+  - Stateful application
+  - Backup and restore
+  - HPE GreenLake for Private Cloud Enterprise
+  - hpe-greenlake-for-private-cloud-enterprise
+---
+<style> li { font-size: 27px; line-height: 33px; max-width: none; } </style>
+
+### Overview
+
+
+
+[HPE GreenLake for Private Cloud Enterprise: Containers](https://www.hpe.com/us/en/greenlake/containers.html) ("containers service"), one of the HPE GreenLake cloud services available on the HPE GreenLake for Private Cloud Enterprise, allows customers to create a Kubernetes (K8s) cluster, view details about existing clusters, and deploy containerized applications to the cluster. It provides an enterprise-grade container management service using open source K8s.  
+
+In this blog post, I discuss first the persistent volumes and volume snapshots in K8s. Then I describe the Container Storage Interface (CSI) and HPE CSI driver for K8s in HPE GreenLake for Private Cloud Enterprise. With a MySQL database instance deployed as a sample stateful application using persistent volume in the cluster, I show the detailed steps to create a volume snapshot of the database as a backup using HPE CSI driver for K8s. Finally I demonstrate how to restore the MySQL database using the created volume snapshot. 

```

Comment:
In the second sentence, add a comma after "Then"

In the third sentence, after the comma, add the word "used" prior to "to create a volume" so it reads "I show the detailed steps used to create a volume..."

In the last sentence, place a comma after "Finally"